bot.py

```diff
@@ -71,22 +71,26 @@ async def on_member_join(member):
     await channel.send(response)
 
 
-@bot.command(name="Welcome")
-async def welcome(ctx):
-    print(f"welcome heard")
-
-    # old/
-    # if message.author == client.user:
-    #    return
-    # if message.content == '!Welcome':
-    #   response = "Howdy"
-    # await message.channel.send(response)
-    # elif message.content == 'raise-exception':
-    # raise discord.DiscordException
-    # old/
-
-    response = f"Howdy {ctx.author}"
-    await ctx.send(response)
+@bot.command(name="roll")
+async def roll(ctx, die: str):
+    try:
+        num_of_dice, die_sides = die.split("d")
+        print(num_of_dice)
+        print(die_sides)
+        int(num_of_dice)
+        int(die_sides)
+    except ValueError:
+        await ctx.send("Error enter ints \nuse command  like !roll 2d20")
+        return
+
+    i = 0
+    while i < int(num_of_dice):
+        roll_result = str(random.randint(1, int(die_sides)))
+        i += 1
+        await ctx.send("Rolling: d" + str(die_sides) +

```

Comment:
could you put the "Rolling: d" + str(die_sides)" outside of the while loop so it only prints once and maybe just say that it is rolling three of that die to reduce chat spam.

Reply:
example:
Rolling 3d20
Rolled: 1
Rolled: 5
Rolled: 6